### DISCUSSION TITLE
Create a plugin in .Net that needs Nuget

### DISCUSSION THREAD

**Discussion by ghost** [Flow-Launcher/Flow.Launcher#1925] [Q&A]
I like to create a plugin that reads data from a database first.
For this I added the Nuget `System.Data.SqlClient.dll`

I added this in my project file to make sure the dll is in the output folder:
`<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>`

But Flow Launcher is not able to find that dll after I copy it to the plugin folder.

Any idea how I can fix this?

**Comment by VictoriousRaptor** [comment]
IIRC our bookmarks plugin uses sqlclient as well. You can take at look at how it works.

**Comment by ghost** [comment]
Thanks, I will take a look.
